config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-x-on-y.yaml

```diff
@@ -1,31 +1,5 @@
 # kubeadm x-on-y tests
 periodics:
-- name: ci-kubernetes-e2e-kubeadm-gce-1-11-on-1-12
```

Comment:
[1]

Reply:
This is not supposed to be removed right? because [1] above says `but this one is OK to be removed in this PR.`
[1]

Reply:
all [1] jobs are OK to be removed.
but `upgrade-1-11-1-12` and `1-11-on-1-12` jobs in general are not OK to be removed when 1.11 goes out of support, as commented here: https://github.com/kubernetes/test-infra/pull/12484#discussion_r281157179